### PR TITLE
[Edit] C++: Methods

### PR DESCRIPTION
**Pull request by KTom101**
### Description

A few errors were found in the C++: Methods code snippets. Fixing so it can run as-is.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/cpp/concepts/methods/methods.md [hunk 1]
```diff
@@ -77,14 +77,14 @@ class Person {
...
+    string set_name(string newName) {
       name = newName;
     }
 
-    void get_name() {
+    string get_name() {
       return name;
     }
-}
+};
```
Comment (avdhoottt) [inline content/cpp/concepts/methods/methods.md:87]: This is giving `Segmentation fault`. It's not fine, right?

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @KTom101, just a small change. Please can you look at it?

**Review by avdhoottt — CHANGES_REQUESTED** [review]
I'll check up this PR.

**Review by avdhoottt — APPROVED** [review]
demo

File: content/cpp/concepts/methods/methods.md [hunk 2]
```diff
@@ -77,14 +77,14 @@ class Person {
 
   public:
     // Defines the method
-    void set_name(string newName) {
+    string set_name(string newName) {
```
Comment (avdhoottt) [inline content/cpp/concepts/methods/methods.md:80]:
```suggestion
    void set_name(string newName) {
```
`set_name` is not returning anything so we should keep it as it is.

**Review by avdhoottt — APPROVED** [review]
Looks good for the second review!!

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Pull request merged**